spec.bs

```diff
@@ -23,38 +23,40 @@ spec: URL; urlPrefix: https://url.spec.whatwg.org/
     text: serialize an integer; url: #serialize-an-integer
 </pre>
 
-<h2 id=urlpattern-class>The {{URLPattern}} class</h2>
+<h2 id=urlpatterns>URL patterns</h2>
 
-A {{URLPattern}} consists of several [=components=], each of which represents a [=/pattern string|pattern=] which could be matched against the corresponding component of a [=/URL=].
+<h3 id=introduction>Intrdocution</h3>
+
+A {{URLPattern}} has an associated [=URL pattern=], which consists of several [=components=], each of which represents a [=/pattern string|pattern=] which could be matched against the corresponding component of a [=/URL=].
```

Comment:
nit: Maybe wait to mention `{{URLPattern}}` until the "the URLPattern class" section, and keep this about describing URL patterns?